homeassistant/components/sensibo/climate.py

```diff
@@ -276,3 +286,25 @@ async def async_assume_state(self, state: str) -> None:
         """Sync state with api."""
         await self._async_set_ac_state_property("on", state != HVACMode.OFF, True)
         await self.coordinator.async_refresh()
+
+    async def async_set_timer(self, state: str, minutes: int | None = None) -> None:
+        """Set or delete timer."""
```

Comment:
Looks like some of these should be VaueError?
https://developers.home-assistant.io/docs/integration_quality_scale_index?_highlight=quality#silver-

Reply:
The problem with ValueError is that it's not giving a response in the UI while HomeAssistantError does.
Probably only the case for these custom services but it's the reason why I used HomeAssistantError instead.

But I will change it as it should be ValueError and test it again and probably raise an issue for the non UI message on ValueError if it's still not showing.